integration_tests/src/main/python/marks.py

```diff
@@ -20,3 +20,4 @@
 incompat = pytest.mark.incompat
 limit = pytest.mark.limit
 qarun = pytest.mark.qarun
+udf = pytest.mark.udf
```

Comment:
Could we rename this to cudf_udf or something like that? Spark already supports a `@udf` for non pandas UDFs and I think it would be best to avoid confusion on it.

Also if your goal is to be able to skip tests if a dependency is not available you can use 

https://docs.pytest.org/en/stable/skipping.html#skipping-on-a-missing-import-dependency

to do that automatically, although that might not work because importing cudf also initializes it on the GPU, which is not really something we want for the main pytest script.

Reply:
Done